### ISSUE TITLE
capture() convenience method not returning expected tuple of Sockets

### ISSUE THREAD

**Issue by Syndaryl** [carson-katri/geometry-script#5]
Blender version: 3.4.0-release
geometry-script version: named-outputs branch

The following code (Based on the example in the documentation) produces a tuple of (String, String) with the values "geometry" and "attribute", not a tuple of (Geometry, (whatever the attribute))

```
@tree("Skin")
def skin():
    # Create a cube
    c = cube()
    # Create a sphere
    sphere = uv_sphere()
    
    return c.capture(position())
```
![image](https://user-images.githubusercontent.com/8820565/206810975-f70a5ea5-fadf-46a5-81a1-edb14ff58af9.png)

**Comment by carson-katri** [comment]
IIRC it returns a `dict` subclass that allows dot access. So perhaps it is not caught by the `isinstance` in #4 and instead should check `issubclass`

**Comment by carson-katri** [comment]
This should be resolved in the latest updates to #4.
